dom.bs

```diff
@@ -1151,22 +1158,42 @@ for discussion).
  <li><p>If <var>target</var> is <var>relatedTarget</var> and <var>target</var> is not
  <var>event</var>'s <a>relatedTarget</a>, then return true.
 
- <li><p>Append (<var>target</var>, <var>targetOverride</var>, <var>relatedTarget</var>) to
- <var>event</var>'s <a for=Event>path</a>.
+ <li><p><a>Append to an event path</a> with <var>event</var>, <var>target</var>,
+ <var>targetOverride</var>, <var>relatedTarget</var>, and false.
 
  <li><p>Let <var>isActivationEvent</var> be true, if <var>event</var> is a {{MouseEvent}} object and
  <var>event</var>'s {{Event/type}} attribute is "<code>click</code>", and false otherwise.
 
  <li><p>Let <var>activationTarget</var> be <var>target</var>, if <var>isActivationEvent</var> is
  true and <var>target</var> has <a for=EventTarget>activation behavior</a>, and null otherwise.
 
+ <li><p>Let <var>slotable</var> be null.
```

Comment:
We should set slotable to target if target is assigned, as we are doing in 11.2, here too.